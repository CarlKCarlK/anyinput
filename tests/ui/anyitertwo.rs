use anyinput::anyinput;

#[anyinput]
pub fn any_str_len(s: AnyIter<AnyString, usize>) -> Result<usize, anyhow::Error> {
    let len = s.len();
    Ok(len)
}

fn main() {}
