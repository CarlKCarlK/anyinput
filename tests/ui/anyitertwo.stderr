error: Expected at exactly one generic parameter.
 --> tests/ui/anyitertwo.rs:4:23
  |
4 | pub fn any_str_len(s: AnyIter<AnyString, usize>) -> Result<usize, anyhow::Error> {
  |                       ^^^^^^^^^^^^^^^^^^^^^^^^^
